src/com/team2502/robot2018/command/autonomous/ingredients/PurePursuitCommand.java

```diff
@@ -41,51 +40,51 @@ public PurePursuitCommand(List<Waypoint> waypoints, Lookahead lookahead, float s
              */
             @Override
             public float getV_rMax()
```

Comment:
You should probably remove these getters if they return `Float.NaN`

Reply:
agreed. kept them due to laziness and time constraints.